app/src/cnm/assets/Questions.csv

```diff
@@ -1,20 +1,20 @@
-17;'MCS STK – Primaquine';'ipc_issueEntry_q_stock_primaquine';;'ipc_issueEntry_q_stock_primaquine';'LLP04MYE6LM';3;0;0;4;17;;26;4;0;;1
-18;'MCS STK – RDTs';'ipc_issueEntry_q_stock_rdt';;'ipc_issueEntry_q_stock_rdt';'wYekTzcJE6c';4;0;0;4;18;;26;4;0;;1
-19;'Code: Not Tested Reason';'ipc_issueEntry_q_not_reason_tested';;'ipc_issueEntry_q_not_reason_tested';'h34mDGMtzuW';2;0;0;1;19;;11;3;1;;1
-20;'Code: Referred';'ipc_issueEntry_q_referred';;'ipc_issueEntry_q_referred';'KV6ssVm0Avk';3;0;0;1;20;;11;3;0;;1
\ No newline at end of file
+1;'Code: Diagnosis Method';'ipc_issueEntry_q_diagnosisMethod';;'ipc_issueEntry_q_diagnosisMethod';'J8SEeQz73ke';1;0;0;1;1;;14;2;0;;1
+2;'Code: Resul RDT';'ipc_issueEntry_q_result_rdt';;'ipc_issueEntry_q_result_rdt';'ZoHcPr1PJ6q';2;0;0;1;2;;11;7;2;;1
+3;'Code: Result Microscope';'ipc_issueEntry_q_result_microscope';;'ipc_issueEntry_q_result_microscope';'ZoHcPr1PJ6q';2;0;0;1;3;;11;7;2;;1
+4;'Code:  OU Village';'ipc_issueEntry_q_village';;'ipc_issueEntry_q_village';'k3gpW1Vm3Ov';3;0;0;2;4;;19;7;0;;0
+5;'Code: OU Tree Village';'ipc_issueEntry_q__tree_village';;'ipc_issueEntry_q__tree_village';'k3gpW1Vm3Ov';4;0;0;2;5;;31;7;0;;0
+6;'Code: Village cords';'ipc_issueEntry_q_village_coords';;'ipc_issueEntry_q_village_coords';'VzOmsEZChev';4;0;0;2;6;;21;7;0;;0
+7;'Code: Age';'ipc_issueEntry_q_patient_age';;'ipc_issueEntry_q_patient_age';'rwgzuR5oI1L';6;0;0;3;7;;6;7;0;;1
+8;'Code: gender Pregnant';'ipc_issueEntry_q_patient_gender_pregnant';;'ipc_issueEntry_q_patient_gender_pregnant';'NoDataElementSexPregnancy';7;0;0;3;8;;20;7;0;;0
+9;'Code: Gender';'ipc_issueEntry_q_patient_gender';;;'uA8F2A6w2Ex';8;0;0;3;9;;21;7;0;;1
+10;'Code: Pregnant';'ipc_issueEntry_q_patient_pregnant';;;'n0ZxPa2rJ58';9;0;0;3;10;;21;7;0;;1
+11;'Code: Diagnosis Type';'ipc_issueEntry_q_diagnosis_type';;'ipc_issueEntry_q_diagnosis_type';'o7mG7zUBGsM';10;0;0;3;11;;32;7;0;;1
+12;'Code: Treatment';'ipc_issueEntry_q_treatment';;'ipc_issueEntry_q_treatment';'rBonfqEGNdS';11;0;0;3;12;;14;7;0;;1
+13;'Code: Other Treatment';'ipc_issueEntry_q_treatment_other';;'ipc_issueEntry_q_treatment_other';'V8q06yRjHqg';12;0;0;3;13;;3;7;0;;1
+14;'MCS STK - ASMQ 100/200 m';'ipc_issueEntry_q_stock_asmq_100_200';;'ipc_issueEntry_q_stock_asmq_100_200';'Y0rvgxJm28P';1;0;0;4;15;;26;4;0;;1
+15;'MCS STK - ASMQ 25/50 mg';'ipc_issueEntry_q_stock_asmq_25_50';;'ipc_issueEntry_q_stock_asmq_25_50';'WA1ouIgQyCr';2;0;0;4;16;;26;4;0;;1
+16;'MCS STK – Primaquine';'ipc_issueEntry_q_stock_primaquine';;'ipc_issueEntry_q_stock_primaquine';'LLP04MYE6LM';3;0;0;4;17;;26;4;0;;1
+17;'MCS STK – RDTs';'ipc_issueEntry_q_stock_rdt';;'ipc_issueEntry_q_stock_rdt';'wYekTzcJE6c';4;0;0;4;18;;26;4;0;;1
+18;'Code: Not Tested Reason';'ipc_issueEntry_q_not_reason_tested';;'ipc_issueEntry_q_not_reason_tested';'h34mDGMtzuW';2;0;0;1;19;;32;2;0;;1
+19;'Code: Referred';'ipc_issueEntry_q_referred';;'ipc_issueEntry_q_referred';'Weypkpeu9Z8';3;0;0;1;20;;32;8;0;;1
```

Comment:
@manuelplazaspalacio there's a conflict with this 2 DEs that has Weypkpeu9Z8 UID. Actually there's no DE in the server with that UID. Can you have a look? Thanks!